config/initializers/pipeline.rb

```diff
@@ -1,4 +1,4 @@
 module IdSeqPipeline
-  S3_SCRIPT_LOC = ENV['IDSEQ_S3_SCRIPT_LOC'] || 's3://czbiohub-infectious-disease/bin/pipeline.py'
-  S3_POSTPROCESS_SCRIPT_LOC = ENV['IDSEQ_POSTPROCESS_S3_SCRIPT_LOC'] || 's3://czbiohub-infectious-disease/bin/postprocess.py'
+  S3_SCRIPT_LOC = ENV['IDSEQ_S3_SCRIPT_LOC'] || "s3://czbiohub-infectious-disease/bin/pipeline-#{Rails.env}.py"
```

Comment:
I think a better way to structure this is to do 
s3://czbiohub-infectious-disease/bin/#{Rails.env}/pipeline.py 
and etc

Reply:
ok